src-js/fontra-webcomponents/src/glyph-cell.js

```diff
@@ -223,13 +220,16 @@ export class GlyphCell extends UnlitElement {
             : html.div(
                 {
                   class: "glyph-shape-placeholder",
+                  ...(this._placeholderDirection
```

Comment:
Suggest to simplify to 
```js
    dir: this._placeholderDirect || "auto",
```
Or, make `this._placeholderDirection` default to `auto` and assign to `dir` unconditionally.